cmd/options/controller_options.go

```diff
@@ -24,15 +24,18 @@ import (
 
 // ControllerOptions contains options and configuration settings for the controller service.
 type ControllerOptions struct {
+	// ExtraTags is a map of tags that will be attached to each dynamically provisioned
+	// resource.
+	ExtraTags map[string]string
 	// ExtraVolumeTags is a map of tags that will be attached to each dynamically provisioned
 	// volume.
 	ExtraVolumeTags map[string]string
```

Comment:
nit: could you add a comment [like this](https://github.com/bertinatto/kubernetes/blob/604569482fc34d010a33339f000e75ecbbf345c6/cmd/kubelet/app/options/options.go#L156)?